### DISCUSSION TITLE
terminal inline assistant?

### DISCUSSION THREAD

**Discussion by cs2018ncsa** [zed-industries/zed#14056] [Feature Requests]
I noticed PR #13731 . Is there already an inline assistant for the terminal in zed or am i misunderstanding what that PR is about? I'm relatively new to following repo activity in GH so maybe I'm missing a description somewhere.

I also don't see any mention of an AI assistant being available for the terminal in docs.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
